.gitignore

```diff
@@ -6,6 +6,7 @@
 
 # Ignore bundler config.
 /.bundle
+.DS_Store
```

Comment:
This should go in a global git ignore, not in the project. https://help.github.com/articles/ignoring-files/#create-a-local-gitignore